lib/actions/FrontendAction.js

```diff
@@ -100,6 +104,7 @@ class FrontendAction {
     switch (action) {
       default:
       case 'start': {
+        utils.generateComponentsJson(this.appSettings)
```

Comment:
I think we should have a test that makes sure the components.json is being generated upon frontend start.